app/containers/ViewQuestion/Bounty.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { APP_FONT } from 'style-constants';
+
+import Container from 'components/Labels/BountyLabel';
+import Span from 'components/Span';
+
+import { convertPeerValueToNumberValue } from 'utils/walletManagement';
+import { getFormattedDate } from 'utils/datetime';
+
+import {
+  BOUNTY_STATUS_ACTIVE,
+  BOUNTY_STATUS_PAID,
+  BOUNTY_STATUS_PENDING,
+  MONTH_3LETTERS__DAY_TIME,
+} from 'utils/constants';
+import {
+  BOUNTY_ACTIVE_CLASSNAME,
+  BOUNTY_PAID_CLASSNAME,
+  BOUNTY_PENDING_CLASSNAME,
+} from './constants';
+
+import BountyPopover from './BountyPopover';
```

Comment:
Move this import to components-imports. Import in groups. It well be easier for understanding for you and your teammates it the future